pymc3/variational/inference.py

```diff
@@ -408,6 +446,78 @@ def from_mean_field(cls, mean_field, gpu_compat=False):
 
     @classmethod
     def from_advi(cls, advi, gpu_compat=False):
+        """
+        Construct FullRankADVI from ADVI
+
+        Parameters
+        ----------
+        advi : ADVI
+
+        Flags
+        -----
+        gpu_compat : bool
+            use GPU compatible version or not
+
+        Returns
+        -------
+        FullRankADVI
+        """
         inference = cls.from_mean_field(advi.approx, gpu_compat)
         inference.hist = advi.hist
         return inference
+
+
+def approximate(n=10000, local_rv=None, method='advi', model=None, **kwargs):
```

Comment:
I think this is great. I just wonder if we can come up with a better name. Technically, MCMC is also approximating the posterior. This is really just performing a type of approximation called variational inference. The correct analog to `sample` would be `optimize`, but that does seem a bit misleading (or maybe not?). Thoughts?